index.d.ts

```diff
@@ -779,29 +787,118 @@ declare namespace Moleculer {
 		 * @param logger the broker logger instance
```

Comment:
The `logger` parameter was eliminated but the jsdoc was left behind.  It looks like there is no `logger` parameter so the jsdoc should be removed.